### ISSUE TITLE
Musesounds playback does not reproduce "mute" on triangle

### ISSUE THREAD

**Issue by Gerardogag** [musescore/MuseScore#20640]
### Issue type

Muse Sounds bug

### Bug description

When writing notes for the triangle and using musesounds the normal notes work fine but, when writing "mute" notes (x head) the sound reproduced is the same as the one produced with the regular note head, and not the muted sound

### Steps to reproduce

1. Open a score that utilizes the triangle instrument 
2. Use the triangle sound from musesounds
3. Write muted triangle notes
4. Begin playback 

### MuseScore Version

4.2

### Regression

Yes, this used to work in a previous version of MuseScore 4.x

### Operating system

Windows 10

**Comment by Gerardogag** [comment]
Interesting, hopefully something can be done so musesounds can recognize the x notehead again, since it worked perfectly in 4.1

**Comment by MarcSabatella** [comment]
Indeed, to be clear: this is almost certainly a recent regression.  The triangle definition includes two notes on the palette, one with an x notehead (shortcut B) for mute, one with a normal notehead (shortcut A) for open.  When you *enter* these notes, they play correctly, and also if you select one in the score.  But on playback, they both play open when using Muse Sounds.  It works correctly in MS Basic.

[Screen recording 2024-01-19 9.27.10 AM.webm](https://github.com/musescore/MuseScore/assets/1799936/33e75194-a3c4-4eba-959d-b0bc74192947)

It's the same *currently* if you test in 4.1.1, but I assume that's because it is using the same version of Muse Sampler as 4.2.1, and that there was a recent regression in Muse Sampler that broke this, not a change in MuseScore itself.
